autoinstaller/install.sh

```diff
@@ -237,9 +518,8 @@ VAI_main() {
     VAI_print_step "Configuring installer"
     VAI_configure_autoinstall
 
-    VAI_print_step "Configuring disk using scheme 'Atomic'"
-    VAI_partition_disk
-    VAI_format_disk
+    VAI_print_step "Configuring disk using recipe defined in config"
```

Comment:
perhaps just an additional variable that can be used to name the scheme.  That way if you have multiple files in a boot menu there is more helpful information available as the system installs.  It would be a manually set field, but would make it easier to see what is going on if there are multiple schemes across multiple config files.